doc/boot_arguments.md

```diff
@@ -15,6 +15,22 @@ See [URL handling in the
 installer](https://github.com/yast/yast-installation/blob/master/doc/url.md) to find more details
 about the supported URLs.
 
+## Custom Installation URL Configuration
+
+You can override the default `installation_url` set in the product files [here](https://github.com/openSUSE/agama/tree/master/products.d) by passing the `agama.install_url` parameter as a boot option in the bootloader.
```

Comment:
Please, I would try to use the same name for the parameter, either `install_url` or `installation_url` but the same.